deltacat/utils/ray_utils/RetryHandler/NonRetryableExc.py

```diff
@@ -0,0 +1,7 @@
+class NonRetryableExc(RuntimeError):
```

Comment:
python module names does not follow camel case generally. So, this file can be renamed to `non_retryable_error.py` and the module as `retry_handler`.